sunpy/time/time.py

```diff
@@ -187,16 +185,13 @@ def parse_time(time_string, time_format='', **kwargs):
 
         time_string_parse_format = kwargs.pop('_time_string_parse_format', None)
         if time_string_parse_format is not None:
-            try:
```

Comment:
I think we needed this. I can't remember why now, but I think it was database related.
I would change the `except:` statement to be `except Exception:` which is still very Pokemon, but at least it's a smidge better.